diffsims/generators/rotation_list_generators.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2019 The diffsims developers
+#
+# This file is part of diffsims.
+#
+# diffsims is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# diffsims is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with diffsims.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Provides users with a range of gridding functions
+"""
+
+import numpy as np
+
+from diffsims.utils.rotation_conversion_utils import Euler
+from diffsims.utils.fundemental_zone_utils import get_proper_point_group_string, reduce_to_fundemental_zone
+from diffsims.utils.gridding_utils import create_linearly_spaced_array_in_rzxz,rotate_axangle, \
+                                          _create_advanced_linearly_spaced_array_in_rzxz, \
+                                          _get_rotation_to_beam_direction
+
+
+def _returnable_eulers_from_axangle(grid,axis_convention,round_to):
+    """ Short snippet that need not be copy pasted across two functions """
+    eulers = grid.to_Euler(axis_convention=axis_convention)
+    rotation_list = eulers.to_rotation_list(round_to=round_to)
+    return rotation_list
+
+def get_fundemental_zone_grid(space_group_number, resolution):
+    """
+    Creates a rotation list for the rotations within the fundemental zone of a given space group.
+
+    Parameters
+    ----------
+    space_group_number : int
+        Between 1 and 230
+
+    resolution : float
+        The 'resolution' of the grid (degrees)
```

Comment:
Can we specify what this means more precisely? Is it the misorientation between points in the grid?